dwave/cloud/client.py

```diff
@@ -142,6 +143,40 @@ class Client(object):
             Problem status is polled with exponential back-off schedule.
             Maximum back-off period is limited to ``poll_backoff_max`` seconds.
 
+        http_retry_total (int, default=10):
+            Total number of retries of failing idempotent HTTP requests to
+            allow. Takes precedence over other counts.
+            See ``total`` in :class:`~urllib3.util.retry.Retry` for details.
+
+        http_retry_connect (int, default=None):
+            How many connection-related errors to retry on.
+            See ``connect`` in :class:`~urllib3.util.retry.Retry` for details.
+
+        http_retry_read (int, default=None):
+            How many times to retry on read errors.
+            See ``read`` in :class:`~urllib3.util.retry.Retry` for details.
+
+        http_retry_redirect (int, default=None):
+            How many redirects to perform.
+            See ``redirect`` in :class:`~urllib3.util.retry.Retry` for details.
+
+        http_retry_status (int, default=None):
+            How many times to retry on bad status codes.
+            See ``status`` in :class:`~urllib3.util.retry.Retry` for details.
+
+        http_retry_backoff_factor (float, default=0.01):
+            A backoff factor to apply between attempts after the second try.
+            Sleep between retries, in seconds::
+
+                {backoff factor} * (2 ** ({number of total retries} - 1))
```

Comment:
is `number of total retries == http_retry_total` ?

Reply:
Because the descriptions refer to urllib3.util.retry.Retry, I found it unclear whether the `default=None` here is equivalent to the `Set to 0` used there or whether further downstream a different default is being set.

Reply:
No, `{number of total retries}` is number of retries at the moment of backoff period calculation.

Whenever some of these parameters is set to `None`, the default value is used. Just like in urllib3, except they have a different default value for `backoff_factor` (0) then us (0.01).